multisense_ros/include/multisense_ros/camera.h

```diff
@@ -77,7 +77,7 @@ class Camera {
     void groundSurfaceCallback(const crl::multisense::image::Header& header);
     void groundSurfaceSplineCallback(const crl::multisense::ground_surface::Header& header);
 
-    void borderClipChanged(const BorderClip &borderClipType, double borderClipValue);
+    void borderClipChanged(const BorderClip& borderClipType, double borderClipValue);
```

Comment:
are these non-ptp changes tracking upstream?
if not, i wouldn't make them here, just to make it easier to merge changes later